tests/v0_tests/test_image_chunking.py

```diff
@@ -16,7 +16,7 @@ class TestImageChunking(MarqoTestCase):
     def setUp(self) -> None:
         client_0 = Client(**self.client_settings)
         
-        self.index_name = 'image-chunk-test'
```

Comment:
As below, I assume these don't change the tests.

Reply:
Following on from above comment, if we have class defined indexes used. we could then just reference `self.index_name  = TestImageChunking.generic_test_index_name` or `self.index_name  = TestImageChunking.generic_test_index_name[-1]`